fitwsarah_frontend/src/components/clientProfile/setting.js

```diff
@@ -2,56 +2,171 @@
 import React, { useEffect, useState } from 'react';
 import { useAuth0 } from '@auth0/auth0-react';
 import "../../css/style.css";
-import { Container, Row, Col } from 'react-bootstrap';
+import {Container, Row, Col, Button, Form} from 'react-bootstrap';
 import NavNotLoggedIn from "../../components/navigation/NotLoggedIn/navNotLoggedIn";
 import NavLoggedIn from "../../components/navigation/loggedIn/navLoggedIn";
 import FooterNotLoggedIn from "../../components/footer/footerNotLoggedIn/footerNotLoggedIn";
 import ProfileSideBar from "../../components/clientProfile/profile";
-const Settings = () => {
-    const { isAuthenticated, getAccessTokenSilently } = useAuth0();
+import {useGetAccessToken} from "../authentication/authUtils";
+import configData from "../../config.json";
+import Sidebar from "../../views/ProfilePage/SideBar";
+function Settings() {
+    const {isAuthenticated, user} = useAuth0();
+    const [accessToken, setAccessToken] = useState(null);
     const [profile, setProfile] = useState(null);
+    const [appointments, setAppointments] = useState([]);
+    const [profilePicUrl, setProfilePicUrl] = useState('');
```

Comment:
Why do we need to have a state for images?

Reply:
I need a state for images because if u look below I used a use effect to verify if the user is valid its gonna call the state setprofilepic and set the users profile picture.